docs/can-guides/commitment/migrating_to_3.0.md

```diff
@@ -1,54 +1,23 @@
 @page migrate-3 Migrating to CanJS 3
 @parent guides/commitment 2
 @templateRender <% %>
+@outline 0
```

Comment:
I’ll add the description. I thought the outline looked out of place because it’s summarized in the text below.